spec/services/omni_auth_service_spec.rb

```diff
@@ -0,0 +1,20 @@
+require 'rails_helper'
+
+RSpec.describe OmniAuthService do
```

Comment:
The structure of the service spec should be very close to the example below. First `describe` is a service class. Next goes the `describe`s of service public methods. Next goes the `context`s (conditions).
```ruby
RSpec.describe OmniAuthService do
  describe '#user' do
    context 'when ...' do
    end

    context 'when ...' do
    end
  end

  describe '#another_method' do
     ...
  end
end
```